index.html

```diff
@@ -0,0 +1,1249 @@
+                        <button id="toast-5" class="show-toast btn btn-danger">error</button>
+
+                        <div class="toast bg-warning toast-6">
+                            <div class="toast-header">
+                                <small>10 min ago</small>
+                            </div>
+                            <div class="toast-footer">
+                                Last day to get this item at 999
+                            </div>
+                        </div>
+                        <button id="toast-6" class="show-toast btn btn-warning">warning</button>
+                    </div>
+                    <iframe
+                        src="https://carbon.now.sh/embed?bg=rgba%2874%2C118%2C159%2C1%29&t=seti&wt=none&l=auto&ds=true&dsyoff=20px&dsblur=68px&wc=true&wa=false&pv=56px&ph=56px&ln=false&fl=1&fm=Hack&fs=14px&lh=133%25&si=false&es=2x&wm=false&code=%253Cdiv%2520class%253D%2522toast%2520bg-success%2520toast-4%2522%253E%250A%2520%2520%253Cdiv%2520class%253D%2522toast-header%2522%253E%250A%2520%2520%2520%2520%2520%253Csmall%253E5%2520min%2520ago%253C%252Fsmall%253E%250A%2520%2520%253C%252Fdiv%253E%250A%2520%2520%253Cdiv%2520class%253D%2522toast-footer%2522%253E%250A%2520%2520%2520%2520%2520Someone%2520viewed%2520your%2520profile%250A%2520%2520%253C%252Fdiv%253E%250A%253C%252Fdiv%253E%250A%253Cbutton%2520id%253D%2522toast-4%2522%2520class%253D%2522show-toast%2520btn%2520btn-success%2522%253Esuccess%253C%252Fbutton%253E%250A%250A%253Cdiv%2520class%253D%2522toast%2520bg-danger%2520toast-5%2522%253E%250A%2520%2520%2520%253Cdiv%2520class%253D%2522toast-header%2522%253E%250A%2520%2520%2520%2520%2520%253Csmall%253E15%2520min%2520ago%253C%252Fsmall%253E%250A%2520%2520%2520%253C%252Fdiv%253E%250A%2520%2520%2520%253Cdiv%2520class%253D%2522toast-footer%2522%253E%250A%2520%2520%2520%2520%2520%2520You%2520plan%2520has%2520been%2520expired%250A%2520%2520%253C%252Fdiv%253E%250A%253C%252Fdiv%253E%250A%253Cbutton%2520id%253D%2522toast-5%2522%2520class%253D%2522show-toast%2520btn%2520btn-danger%2522%253Eerror%253C%252Fbutton%253E%250A%250A%253Cdiv%2520class%253D%2522toast%2520bg-warning%2520toast-6%2522%253E%250A%2520%2520%2520%253Cdiv%2520class%253D%2522toast-header%2522%253E%250A%2520%2520%2520%2520%2520%253Csmall%253E10%2520min%2520ago%253C%252Fsmall%253E%250A%2520%2520%2520%253C%252Fdiv%253E%250A%2520%2520%2520%253Cdiv%2520class%253D%2522toast-footer%2522%253E%250A%2520%2520%2520%2520%2520Last%2520day%2520to%2520get%2520this%2520item%2520at%2520999%250A%2520%2520%2520%253C%252Fdiv%253E%250A%253C%252Fdiv%253E%250A%253Cbutton%2520id%253D%2522toast-6%2522%2520class%253D%2522show-toast%2520btn%2520btn-warning%2522%253Ewarning%253C%252Fbutton%253E"
+                        " class="iframe-container" sandbox="allow-scripts allow-same-origin">
+                    </iframe>
+                </div>
+            </section>
+
+            <!-- Toast/Snackbar -->
+
+        </div>
+
+        <footer class="footer">

```

Comment:
footer links for socials should open in new tab

Reply:
done